misc/dbt-materialize/tests/adapter/test_custom_materializations.py

```diff
@@ -62,15 +64,16 @@ def test_custom_materializations(self, project):
         # run models
         results = run_dbt(["run"])
         # run result length
-        assert len(results) == 9
+        assert len(results) == 10
         # re-run models to ensure there are no lingering errors in recreating
         # the materializations
         results = run_dbt(["run"])
         # re-run result length
-        assert len(results) == 9
+        assert len(results) == 10
         # relations_equal
         check_relations_equal(
-            project.adapter, ["test_materialized_view", "test_view_index"]
```

Comment:
Noticed that this test was a leftover from the old definition of materialized views (which were equivalent to a view with a default index on it); repurposing it, since in the context of dbt we want the `materializedview` and `table` materializations to behave the same.